StreamingAssets.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
should this file be here? StreamingAssets?

Reply:
Good point - I will remove it, THanks!